text/0000-abandoning-crates.md

```diff
@@ -0,0 +1,114 @@
+- Feature Name: Process for Abandoning Crates
+- Start Date: 2016-11-16
+- RFC PR:
+- Rust Issue:
+
+# Summary
+[summary]: #summary
+Open source development is a fast moving target, with people creating and abandoning their projects
+all the time. However, the project names on crates.io are static and can never be changed.
+
+There is currently no way for admins to forcefully transfer ownership of a
+crate, and this is good. Rust should strive to avoid some of the drama that has
+inflicted other communities, such as the notorious
+[left-pad](http://www.theregister.co.uk/2016/03/23/npm_left_pad_chaos/) incident
```

Comment:
Yes, you are right. However it was Koçulu (or Azer? I'm not sure which name to use) who was wronged in that case. I will update